src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -130,37 +130,37 @@
     <value>Use generic event handler instances</value>
   </data>
   <data name="UseGenericEventHandlerInstancesForDelegateMessage" xml:space="preserve">
-    <value>Remove '{0}' and replace its usage with a generic EventHandler, for example EventHandler&lt;T&gt;, where T is a valid EventArgs</value>
+    <value>Remove '{0}' and replace its usage with a generic 'EventHandler', for example 'EventHandler&lt;T&gt;', where 'T' is a valid 'EventArgs'</value>
   </data>
   <data name="UseGenericEventHandlerInstancesForDelegateDescription" xml:space="preserve">
     <value>A type contains a delegate that returns void, whose signature contains two parameters (the first an object and the second a type that is assignable to EventArgs), and the containing assembly targets Microsoft .NET Framework?2.0.</value>
   </data>
   <data name="UseGenericEventHandlerInstancesForEventMessage" xml:space="preserve">
-    <value>Change the event '{0}' to replace the type '{1}' with a generic EventHandler, for example EventHandler&lt;T&gt;, where T is a valid EventArgs</value>
+    <value>Change the event '{0}' to replace the type '{1}' with a generic 'EventHandler', for example 'EventHandler&lt;T&gt;', where 'T' is a valid 'EventArgs'</value>
   </data>
   <data name="UseGenericEventHandlerInstancesForEventDescription" xml:space="preserve">
     <value>A delegate that handles a public or protected event does not have the correct signature, return type, or parameter names.</value>
   </data>
   <data name="UseGenericEventHandlerInstancesForEvent2Message" xml:space="preserve">
-    <value>Change the event '{0}' to use a generic EventHandler by defining the event type explicitly, for e.g. Event MyEvent As EventHandler(Of MyEventArgs).</value>
+    <value>Change the event '{0}' to use a generic 'EventHandler' by defining the event type explicitly, for example, 'Event MyEvent As EventHandler(Of MyEventArgs)'.</value>
```

Comment:
Why is this using VB-specific syntax? 😨 
The analyzer seems applicable to both languages?
https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/UseGenericEventHandlerInstances.cs#L22

Reply:
Yes, we should fix that. Additionally, we should also not be giving examples in the diagnostic message. Instead the example should go into the diagnostic description.

Reply:
@mavasani I'm going to open a separate issue tracking this to first confirm this is reachable in C#, and see how can we make the message read well for both languages.

FYI: the message here was ending in a period while it shouldn't. The analyzer didn't catch that initially because it thought it's a muli-sentence message (because of `e.g.` - the analyzer treated `.` as end of sentence, hence decided it's multi-sentence). I'm not sure this is fixable.

```suggestion
    <value>Change the event '{0}' to use a generic 'EventHandler' by defining the event type explicitly, for example, 'Event MyEvent As EventHandler(Of MyEventArgs)'</value>
```